src/main/java/io/vertx/core/http/impl/Http1xServerRequest.java

```diff
@@ -117,35 +120,34 @@ HttpRequest nettyRequest() {
     }
   }
 
-  private InboundBuffer<Object> pendingQueue() {
-    if (pending == null) {
-      pending = new InboundBuffer<>(context, 8);
-      pending.drainHandler(v -> conn.doResume());
-      pending.handler(buffer -> {
-        if (buffer == InboundBuffer.END_SENTINEL) {
-          onEnd();
-        } else {
-          onData((Buffer) buffer);
-        }
-      });
-    }
-    return pending;
-  }
-
-  void handleContent(Buffer buffer) {
-    InboundBuffer<Object> queue;
+  boolean handleContent(HttpContent content) {
+    boolean last = content instanceof LastHttpContent;
```

Comment:
this is going to trigger the type pollution problem here...
Please check first if `msg == LastHttpContent.EMPTY_LAST_CONTENT` or check how we've done elsewhere